lib/facter/mysql_version.rb

```diff
@@ -0,0 +1,8 @@
+Facter.add("mysql_version") do
+  setcode do
+    mysql_ver = Facter::Util::Resolution.exec('mysql --version')
+    if mysql_ver
+      mysql_ver.match(/\d+\.\d+\.\d+/)[0]
```

Comment:
this gives us 10.0.19 for mariadb-client-10.0, and 5.6.24 for mysql-client-core-5.6

i wonder what we can do with that now…